sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -2198,6 +2201,27 @@ public void setIgnoredCheckIns(final @Nullable List<String> ignoredCheckIns) {
     }
   }
 
+  @ApiStatus.Experimental
+  public @Nullable List<String> getIgnoredSpanOrigins() {
+    return ignoredSpanOrigins;
+  }
+
+  @ApiStatus.Experimental
+  public void setIgnoredSpanOrigins(final @Nullable List<String> ignoredSpanOrigins) {
+    if (ignoredSpanOrigins == null) {
+      this.ignoredSpanOrigins = null;
+    } else {
+      @NotNull final List<String> filtered = new ArrayList<>();
+      for (String origin : ignoredSpanOrigins) {
+        if (!origin.isEmpty()) {
```

Comment:
Could `ignoredSpanOrigins` potentially contain null values? Maybe add a null check to be safe.

` if (origin != null && !origin.isEmpty())`